source/aggregation.txt

```diff
@@ -0,0 +1,210 @@
+.. _kotlin-sync-aggregation:
+
+====================================
+Transform Your Data with Aggregation
+====================================
```

Comment:
Q: I know this is part of the TOC we are adhering to, but would it make sense to just call this page "Transform Your Data"?

Reply:
I think it's helpful to leave the Aggregation in the title here since it puts a name on the subject of the page tree. "Transform Your Data" is helpful for those unfamiliar with driver concepts, while the "Aggregation" part of the name is helpful for those who are familiar.